Library/Homebrew/dev-cmd/typecheck.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  module_function
+
+  def typecheck_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `typecheck`
+
+        Check for typechecking errors using Sorbet.
+      EOS
+      switch "-q", "--quiet",
+             description: "Silence all non-critical errors."
+      switch "--update-definitions",
+             description: "Update Tapioca gem definitions of recently bumped gems"
+      switch "--fail-if-not-changed",
```

Comment:
Can we make this `--fail-if-changed`? The double negative is kind of confusing.

Reply:
No, this matches `-update-license-data`.

Reply:
Or at least: yes, it could but both would need to be updated (which is out of scope for this PR specifically).